src/@utils/wallet/index.ts

```diff
@@ -51,9 +51,13 @@ export const connectKitTheme = {
   '--ck-secondary-button-border-radius': 'var(--border-radius)'
 }
 
-export function accountTruncate(account: string): string {
+export function accountTruncate(
+  account: string,
+  begin: number = 6,
+  end: number = 38
```

Comment:
nitpick(non-blocking): Could you add a comment explaining these default magic numbers as well?